src/geometry/Point.cpp

```diff
@@ -0,0 +1,10 @@
+#include <Point.h>
+#include <cmath>
```

Comment:
```suggestion
#include <point.h>
#include <cmath>
```

vamo tentar deixar nome de lib em minúsculo, e só o nome das classes em si começando com a primeira letra maiúscula

Reply:
Feito @kinhosz

Reply:
Renomei tanto o header file quanto o source code

`Point.cpp` -> `point.cpp`
`Point.h` -> `point.h`